playbooks/common/openshift-master/post_config.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Configure Node Labels
+  hosts: oo_masters_to_config
+  roles:
+    - openshift_label_nodes
```

Comment:
I am fine with putting it in register nodes, in which case that only runs on the first master ans solves the HA master thing.